packages/transaction_signing_gateway/test/mocks/test_memory_store.dart

```diff
@@ -4,6 +4,12 @@ import 'package:dartz/dartz.dart';
 class TestMemoryStore implements PlainDataStore, SecureDataStore {
   final Map<String, String?> _values = {};
 
+  @override
+  Future<Either<CredentialsStorageFailure, bool>> clearAllData() async {
+    _values.clear();
```

Comment:
Should be wrapped in try catch

Reply:
will do

Reply:
actually, no need to wrap it as `clear` does not throw anything